addons/mail/static/src/core/web/follower_subtype_dialog.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates xml:space="preserve">
     <t t-name="mail.FollowerSubtypeDialog">
-        <Dialog bodyClass="'d-flex flex-column'" contentClass="'o-mail-FollowerSubtypeDialog'" size="'sm'" title="title">
+        <Dialog t-if="props.follower.partner" bodyClass="'d-flex flex-column'" contentClass="'o-mail-FollowerSubtypeDialog'" size="'sm'" title="title">
```

Comment:
I think just `props.follower` is enough

Reply:
`props.follower` throws error while `props.follower.partner` does not.

Reply:
Ok, it fixes the issue with minimal diff so it's good ;)